src/main/java/net/glowstone/entity/passive/GlowCow.java

```diff
@@ -19,6 +21,19 @@ public GlowCow(Location location) {
     }
 
     @Override
+    public void kill() {
+        super.kill();
+
+        Random r = new Random();
+
+        int beefCnt = r.nextInt(3);
+        int leatherCnt = r.nextInt(2);
+
+        getWorld().dropItemNaturally(getLocation(), new ItemStack(Material.RAW_BEEF, beefCnt));
```

Comment:
Can you please clarify the differences between a block and an entity?

For both classes some items are dropped after death/destruction.
Maybe we can do an Interface e.g. IDropsItemsAfterDead (.net Interfaces start with capital i, same in java?)